docs/docs/hive.md

```diff
@@ -34,6 +34,32 @@ Iceberg compatibility with Hive 2.x and Hive 3.1.2/3 supports the following feat
 !!! warning
     DML operations work only with MapReduce execution engine.
 
+The HiveCatalog supports the following additional features with Hive version 4.0.0 and above:
```

Comment:
Is this HiveCatalog, or Hive?
`HiveCatalog`, and `Catalog` in Iceberg context refers to the implementations of the Catalog interface, which allows atomic changes of the table.

I suspect that this paragraph refers to the Hive Engine itself